compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensionsOps.td

```diff
@@ -642,4 +642,30 @@ def GpuDistributeSharedMemoryCopyOp :
   }];
 }
 
+def CreateTileSizesOp :
+  Op<Transform_Dialect, "iree.create_tile_sizes",
+    [MemoryEffectsOpInterface,
+     TransformOpInterface,
+     TransformEachOpTrait,
+     ParamProducerTransformOpTrait]> {
+  let description = [{
+    Create param of tile sizes based on the matmul sizes.
+  }];
+
+  let arguments = (ins TransformHandleTypeInterface:$target);
+  let results = (outs TransformParamTypeInterface:$tile_size0,
+                      TransformParamTypeInterface:$tile_size1);
```

Comment:
If I'm understanding correctly, the tile sizes returned here are `workgroup_level` and `warp_level`? I would rename `tile_size0` -> `workgroup_tile_sizes` and `tile_size1` -> `problem_specific_sizes` or something like that.